scripts/common.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+
+
+function getRdsStatus() {
+    for attempt in {1..10}
+    do
+        rds_status=$(aws rds describe-db-instances --db-instance-identifier ${RDS_INSTANCE} --query 'DBInstances[0].DBInstanceStatus' --output text)
+        if [[ ! -z ${rds_status} ]]; then
+            break
+        fi
+    done
+    echo ${rds_status}
+}
+
+function waitForInstanceAvailable() {
+    for attempt in {1..10}
+    do
+        current_status=$(getRdsStatus)
+        if [[ "$current_status" == "available" ]]; then
```

Comment:
This will work fine but I think it's good practice to always write variables with the curly braces when quoting them.